_data/footer.yml

```diff
@@ -7,4 +7,7 @@
     link: https://github.com/openfreeenergy
     new_window: True
     social_icon: GitHub  # we'll need to get this from another source
-
+  - name: Publications
```

Comment:
Not sure about "Publications" as the label on this one. Here's the current GitHub and Twitter rendering:
<img width="411" alt="image" src="https://github.com/OpenFreeEnergy/openfreeenergy.github.io/assets/8178546/61c36bc9-df8a-4254-85e1-3eccccd24e43">

I'm not sure that having "Zenodo: Publications" fits the pattern. Additionally, "Publications" in my mind, in this context, suggests peer-reviewed academic publications. While those may also be on Zenodo, most of what we'll have is other miscellaneous documents (talks, posters, etc.)

Reply:
we could go with talks, posters etc. but somwhere it would be cool if we could collect any kind of publication we do (Poster, Talks and Papers) :)

Reply:
which actually brings up the question, can we link preprints or journal publications to zenodo? rather not right?

Reply:
that might be something for the future, like having a rolling list of all publications we put out there :)

Reply:
I think zenodo is a platform for gathering things that aren't deposited anywhere else to assign them a DOI.  Preprints/publications don't really need this treatment.

Reply:
I just changed the name to the Zenodo group name